sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/dsl/package.scala

```diff
@@ -83,7 +83,10 @@ package object dsl {
     def >= (other: Expression): Predicate = GreaterThanOrEqual(expr, other)
     def === (other: Expression): Predicate = EqualTo(expr, other)
     def <=> (other: Expression): Predicate = EqualNullSafe(expr, other)
```

Comment:
Ok, so it seems it is a well-established operator. Unfortunately <=> it also has a slightly lower precedence than =!= and ===, but since it is something known and I see no suitable alternative, I guess its best to live with it